inbc-program/inbc/parser.py

```diff
@@ -284,18 +292,19 @@ def sota(args) -> str:
                 '<repo>remote</repo>' +
                 '</header>' +
                 '<type><sota>' +
-                '<cmd logtofile="y">update</cmd>' +
-                '{0}' +
+                '<cmd logtofile="y">{0}</cmd>' +
+                '{1}' +
```

Comment:
I don't think we need to add the update to command to INBC since it's always update.  I'd leave this hardcoded and remove the command from INBC as an option.